### DISCUSSION TITLE
Update to baikal v0.10.0 and include Home Assistant Fix

### DISCUSSION THREAD

**Discussion by update-freak** [ckulka/baikal-docker#226] [General]
Hi developer,

are you planning to update to v0.10.0?
By doing this maybe also include the Home Assistant Fix which is described here: https://github.com/sabre-io/dav/issues/1318#issuecomment-1872627379
Thank you.

**Comment by kiwinh0** [comment]
out v0.10.1

**Reply by mcmikemn** [reply]
The [Home Assistant Fix](https://github.com/ckulka/baikal-docker/issues/231) is part of v0.10.1? I have v0.10.1 installed and:
```
    environment:  
      APPLY_HOME_ASSISTANT_FIX: "true"
```
in `docker-compose.yaml`, but there is no mention in the logs of:
```
30-apply-home-assistant-fix.sh: info: Applying Home Assistant fix
30-apply-home-assistant-fix.sh: info: Applied Home Assistant fix
```

It looks like the fix was [merged to master last month](https://github.com/ckulka/baikal-docker/pull/241), _after_ v0.10.1 was released on November 17, 2024. Am I looking at this incorrectly? Or maybe I applied the fix incorrectly?
